src/components/Layout/Header/style.module.less

```diff
@@ -89,6 +89,21 @@
       }
     }
   }
+
+  button {
+    background: none;
+    color: inherit;
+    border: none;
+    padding: 0;
+    font: inherit;
+    cursor: pointer;
+    outline: inherit;
```

Comment:
This breaks the a11y, buttons has not :focus state anymore. Did it end up there by accident? For me there is no visible difference when just deleted.

Reply:
Ok, will remove this and will split the .less file :)

Reply:
do you mean just the "outline" attribute? @timurc

Reply:
Fixed